scripts/install-chef-suse.sh

```diff
@@ -344,19 +344,28 @@ EOF
     exit 1
 fi
 
-if ! (systemctl -q is-active crowbar-init.service && reset_crowbar ); then
-
-    cat <<EOF | pipe_show_and_log
+if ! (reset_crowbar 2> /dev/null); then
```

Comment:
Do we really need to hide the output?

Reply:
Regardless of whether we do or don't need to hide it please come to a consensus. For according to QA this _does_ fix a problem... :-)

Reply:
Well, not really. I can unhide it :smile: